packages/slice-machine/components/ScreenshotChangesModal/index.tsx

```diff
@@ -124,23 +127,22 @@ const ScreenshotChangesModal = ({
   slices: ComponentUI[];
   defaultVariationSelector?: SliceVariationSelector;
 }) => {
-  if (slices.length === 0) return null;
-
   const { closeScreenshotsModal } = useSliceMachineActions();
 
   const { isOpen } = useSelector((store: SliceMachineStoreType) => ({
     isOpen: isModalOpen(store, ModalKeysEnum.SCREENSHOTS),
   }));
 
-  const [variationSelector, setVariationSelector] =
-    useState<SliceVariationSelector>(
-      variationSetter(defaultVariationSelector, slices)
-    );
+  const [variationSelector, setVariationSelector] = useState(
+    variationSetter(defaultVariationSelector, slices)
+  );
 
   useEffect(() => {
     setVariationSelector(variationSetter(defaultVariationSelector, slices));
   }, [defaultVariationSelector, isOpen]);
```

Comment:
I thought we added the eslint rule to detect missing hook dependencies. Does it not trigger an error?

Reply:
Did you create the tech debt item (I couldn't find one)? Otherwise I'm worried we'll never come back to this, warnings are usually ignored

Reply:
I'll add a ticket to the board :)